### PR TITLE
markdig fixes - part 3

### PR DESCRIPTION
**Pull request by mairaw**
Hide whitespace changes for easier diff

/cc @scottaddie

### CODE REVIEW

File: docs/framework/wcf/feature-details/deploying-an-internet-information-services-hosted-wcf-service.md [hunk 1]
```diff
@@ -4,99 +4,102 @@ ms.date: "03/30/2017"
...
+
+## Configure the WCF Service
+IIS-hosted WCF services store their configuration in the applications Web.config file. IIS-hosted services use the same configuration elements and syntax as WCF services hosted outside of IIS. However, the following constraints are unique to the IIS hosting environment:
+
+- Base addresses for IIS-hosted services.
+
+- Applications hosting WCF services outside of IIS can control the base address of the services they host by passing a set of base address URIs to the <xref:System.ServiceModel.ServiceHost> constructor or by providing a [\<host>](../../../../docs/framework/configure-apps/file-schema/wcf/host.md) element in the service’s configuration. Services hosted in IIS do not have the ability to control their base address; the base address of an IIS-hosted service is the address of its .svc file.
+
+### Endpoint Addresses for IIS-Hosted Services
+ When hosted in IIS, endpoint addresses are always considered to be relative to the address of the .svc file that represents the service. For example, if the base address of a WCF service is http://localhost/Application1/MyService.svc with the following endpoint configuration.
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/deploying-an-internet-information-services-hosted-wcf-service.md]: the URL should be fenced here

File: docs/framework/wcf/feature-details/deploying-an-internet-information-services-hosted-wcf-service.md [hunk 2]
```diff
@@ -4,99 +4,102 @@ ms.date: "03/30/2017"
...
+
+```xml
+<endpoint address="anotherEndpoint" .../>
+```
+
+This provides an endpoint that can be reached at `http://localhost/Application1/MyService.svc/anotherEndpoint`.
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/deploying-an-internet-information-services-hosted-wcf-service.md]: Also fence the URL.

File: docs/framework/wcf/feature-details/deploying-an-internet-information-services-hosted-wcf-service.md [hunk 3]
```diff
@@ -4,99 +4,102 @@ ms.date: "03/30/2017"
...
+
+ Similarly, the endpoint configuration element that uses an empty string as the relative address provides an endpoint reachable at http://localhost/Application1/MyService.svc, which is the base address.
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/deploying-an-internet-information-services-hosted-wcf-service.md]: fence the URL.

File: docs/framework/wcf/feature-details/deploying-an-internet-information-services-hosted-wcf-service.md [hunk 4]
```diff
@@ -4,99 +4,102 @@ ms.date: "03/30/2017"
...
+
+```xml
+<endpoint address="" ... />
+```
+
+You must always use relative endpoint addresses for IIS-hosted service endpoints. Supplying a fully-qualified endpoint address (for example, `http://localhost/MyService.svc`) can lead to errors in the deployment of the service if the endpoint address does not point to the IIS-application that hosts the service exposing the endpoint. Using relative endpoint addresses for hosted services avoids these potential conflicts.
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/deploying-an-internet-information-services-hosted-wcf-service.md]: fence the URL.

**Review by rpetrusha — APPROVED** [review]
LGTM, @mairaw. There are a couple of additional URLs that could be fenced. You can merge when you're ready.

File: docs/framework/wcf/feature-details/deploying-an-internet-information-services-hosted-wcf-service.md (hunk 2 above)
Comment (mairaw) [reply docs/framework/wcf/feature-details/deploying-an-internet-information-services-hosted-wcf-service.md]: This one is fenced already!

File: docs/framework/wcf/feature-details/deploying-an-internet-information-services-hosted-wcf-service.md (hunk 4 above)
Comment (mairaw) [reply docs/framework/wcf/feature-details/deploying-an-internet-information-services-hosted-wcf-service.md]: also already fenced no?

**Pull request merged**